x-pack/test/functional/apps/ml/anomaly_detection_result_views/anomaly_explorer.ts

```diff
@@ -151,6 +151,26 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
           await ml.anomaliesTable.assertTableNotEmpty();
         });
 
+        it('should allow filtering by influencer', async () => {
+          const fieldName = testData.expected.influencers[0].field;
+          const fieldValue = testData.expected.influencers[0].labelsContained[0];
+
+          await ml.testExecution.logTestStep(
+            'adds influencer filter by clicking on the influencer add filter button'
+          );
+          await ml.anomalyExplorer.addFilterForInfluencer(fieldName, fieldValue);
+          await ml.testExecution.logTestStep('query bar and table rows reflect filter');
+          await ml.anomalyExplorer.assertQueryBarContent(`${fieldName}:"${fieldValue}"`);
+          await ml.anomaliesTable.assertInfluencersCellsContainFilter(
```

Comment:
Can you add in additional checks to assert the contents of the Top influencers list and that the 'view by' swim lane contains the expected single value - looks like these sort of checks are already in place in other tests in this file.

Reply:
Added in 2f7485f24952fa0b40229d8a7cf42d0f2a4a4661